.evergreen/.evg.yml

```diff
@@ -150,6 +150,8 @@ functions:
         env:
           PRODUCT_NAME: ${product_name}
           PRODUCT_VERSION: ${product_version}
+          PRODUCT_RELEASE_CREATOR: ${author}
+          EVERGREEN_VERSION_ID: ${version_id}
```

Comment:
@vbabanin proposed a great idea to use [Evergreen default expansions](https://docs.devprod.prod.corp.mongodb.com/evergreen/Project-Configuration/Project-Configuration-Files#default-expansions) instead of trying to compute the required information on our own. Implementing this proposal greatly simplified the PR. However, I don't see how to test the new approach for builds triggered by a Git tag. We will see today how that works when releasing 5.1.2.